hedera-node/src/main/java/com/hedera/services/legacy/core/jproto/TxnReceipt.java

```diff
@@ -130,6 +130,7 @@ public int getVersion() {
 	public void serialize(SerializableDataOutputStream out) throws IOException {
 		out.writeNormalisedString(status);
 		out.writeSerializable(exchangeRates, true);
+		//TODO: can use writeSerializable which has support for null
```

Comment:
Since we have created an issue to refactor DomainSerdes class, this TODO can be removed